### PR TITLE
Add manifests

### PR DESCRIPTION
**Pull request by pacospace**

### CODE REVIEW

File: manifests/base/imagestream.yaml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+---
+apiVersion: image.openshift.io/v1
+kind: ImageStream
+metadata:
+  name: aidevsecops-tutorial
+spec:
+  lookupPolicy:
+    local: true
```
Comment (goern) [inline manifests/base/imagestream.yaml:8]: this imag stream has no tag, so there is really now image in here...

**Review by goern — CHANGES_REQUESTED** [review]

File: manifests/base/imagestream.yaml (hunk 1 above)
Comment (pacospace) [reply manifests/base/imagestream.yaml]: I was not sure if we wanted to have `overlays/`! Thanks for review, I added it :) there is no image on quay yet in any case!

/hold

**Review by goern — APPROVED** [review]

File: manifests/base/imagestream.yaml (hunk 1 above)
Comment (harshad16) [reply manifests/base/imagestream.yaml]: we should let `base/imagestream.yaml` have no tags and include an overlay `test` or something, include imagestream with a tag in that. 
In this way, we would be demonstrating a good practice of kustomize base/overlays dir.

**Pull request merged**